operator/manager/rpc/client.go

```diff
@@ -38,6 +39,10 @@ type Client struct {
 	client *retryablehttp.Client
 }
 
+func (s *Client) UploadCard(ctx context.Context, step int64, input *core.CardInput) error {
```

Comment:
this should have a test similar to the other functions in this package.

Reply:
To write tests similar to the ones in the package, id have to write code to hook it up in server.go which is kinda pointless dead code?